callback_plugins/help.py

```diff
@@ -48,29 +47,67 @@ def __init__(self):
+        abridged_result = strip_internal_keys(result._result)
+
+        # remove invocation unless specifically wanting it
+        if not keep_invocation and self._display.verbosity < 3 and 'invocation' in abridged_result:
+            del abridged_result['invocation']
+
+        # remove diff information from screen output
+        if self._display.verbosity < 3 and 'diff' in abridged_result:
+            del abridged_result['diff']
+
+        if 'access_control_allow_headers' in abridged_result:
+            del abridged_result['access_control_allow_headers']
+
+        if 'access_control_allow_methods' in abridged_result:
+            del abridged_result['access_control_allow_methods']
+
+        if 'access_control_allow_origin' in abridged_result:
+            del abridged_result['access_control_allow_origin']
+
+        if 'x_content_type_options' in abridged_result:
+            del abridged_result['x_content_type_options']
+
+        # remove exception from screen output
+        if 'exception' in abridged_result:
+            del abridged_result['exception']
+
+        dumped = ''
+
+        dumpd_tile = '[' + str(result._host.name) + ']: Ansible Failed! ==>\n  '
+        # put changed and skipped into a header line
+        if 'changed' in abridged_result:
+            dumped += 'changed=' + str(abridged_result['changed']).lower() + ' '
+            del abridged_result['changed']
+
+        if 'skipped' in abridged_result:
+            dumped += 'skipped=' + str(abridged_result['skipped']).lower() + ' '
+            del abridged_result['skipped']
+
+        # if we already have stdout, we don't need stdout_lines
+        if 'stdout' in abridged_result and 'stdout_lines' in abridged_result:
+            abridged_result['stdout_lines'] = '<omitted>'
+
+        if abridged_result:
+            dumped += '\n'
+            dumped += yaml.dump(abridged_result, width=1000, Dumper=AnsibleDumper, default_flow_style=False)
+
+        # indent by a couple of spaces
+        dumped = '\n  '.join(dumped.split('\n')).rstrip()
+        return dumpd_tile + dumped + '\n'
 
     def print_help_message(self):
-        self._display.display("Ask for help:", color=C.COLOR_WARN)
-        self._display.display("Contact us: support@pingcap.com", color=C.COLOR_HIGHLIGHT)
+        self._display.display("Ask TiDB User Group for help:", color=C.COLOR_WARN)
         self._display.display(
-            "It seems that you encounter some problems. You can send an email to the above email address, attached with the tidb-ansible/inventory.ini and tidb-ansible/log/ansible.log files and the error message, or new issue on https://github.com/pingcap/tidb-ansible/issues. We'll try our best to help you deploy a TiDB cluster. Thanks. :-)",
+            "It seems that you have encountered some problems, you can describe the operation steps and error information as much as possible on https://asktug.com. We will do our best to help you solve the problem.. Thanks. :-)",
```

Comment:
```suggestion
            "It seems that you have encountered some problem. Please describe your operation steps and provide error information as much as possible on https://asktug.com (in Chinese) or https://stackoverflow.com/questions/tagged/tidb (in English). We will do our best to help solve your problem. Thanks. :-)",
```

Reply:
FYI. `some` can mean `a certain` when you don't know what thing exactly. @liubo0127